axelor-purchase/src/main/resources/views/PurchaseOrderLine.xml

```diff
@@ -135,8 +135,16 @@
 				<panel-related name="purchaseOrderSupplierLineListPanel" field="purchaseOrderSupplierLineList" colSpan="12" form-view="purchase-order-supplier-line-form" grid-view="purchase-order-supplier-line-grid"/>
 			</panel>
 			<panel name="budgetDistributionPanel" title="Budget" if="__config__.app.isApp('budget')" if-module="axelor-supplychain">
+				<field name="companyExTaxTotal" title="Total W.T." colSpan="6" if="__config__.app.getApp('budget').getManageMultiBudget()" readonly="true" hidden="false">
+					<viewer depends="companyExTaxTotal,purchaseOrder.company.currency.symbol">
+						<![CDATA[
+							<dd>{{record.companyExTaxTotal}} {{record.purchaseOrder.company.currency.symbol}}</dd>
```

Comment:
Currency will not display with new record. Use dummy field to get value from parent instead of using record.purchaseOrder..etc